frontend/src/components/ShootTickets/TicketComment.vue

```diff
@@ -38,7 +39,8 @@ const AvatarEnum = {
 
 export default {
   components: {
-    TimeString
+    TimeString,
+    ExternalLink
```

Comment:
remove mixin

Reply:
same, right now the button has a `:href` but it would also possible to use the `ExternalLink` component in the button slot I guess.